src/test/java/parser/InputParserTest.java

```diff
@@ -0,0 +1,46 @@
+package parser;
+
+import command.ByeCommand;
+import command.Command;
+import command.IncorrectCommand;
+import command.MarkCommand;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class InputParserTest {
+    @Test
+    public void parseCommand_bye_success() {
+        assertInstanceOf(ByeCommand.class, InputParser.parseCommand("bye"));
+    }
+
+    @Test
+    public void parseCommand_mark_success() {
```

Comment:
Perhaps you can specify what are the properties of the user input that makes parsing it into a MarkCommand a success, for example (same as other test methods)?

```suggestion
    public void parseCommand_markUserInputHasValidIntegerIndex_success() {
```

Reply:
Noted, thanks for the suggestion.